app/src/main/res/drawable/ripple_with_background.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- have different ripple color on light/dark theme so it's well visible on both -->
```

Comment:
I don't understand this comment, how does this set a different ripple color depending on the theme mode?

Reply:
It's an outdated comment, will update FeelsDankMan